phusion-ruby-fips/Description.md

```diff
@@ -3,10 +3,10 @@
  with Ruby compiled against the FIPS 140-2 compliant [OpenSSL module](https://www.openssl.org/docs/fips.html).  
 This image includes the following packages:
 
-* OpenSSL version: `1.0.2u` build with  FIPS 140-2 compliant OpenSSL module version: `2.0.16`
-* Ruby version: `2.5` compiled against the FIPS 140-2 compliant OpenSSL module
-* Postgres client version: `12-12.2` compiled against the FIPS 140-2 compliant OpenSSL module
-* OpenLDAP version: `2.4.46` build using openssl rather than gnutls and compiled against the FIPS 140-2 compliant OpenSSL module 
-* Bundler version: `2.1.4`
+* OpenSSL version `1.0.2u`: built with  FIPS 140-2 compliant OpenSSL module version `2.0.16`.
+* Ruby version `2.5`: compiled against the FIPS 140-2 compliant OpenSSL module.
+* Postgres client version `12-12.2`: compiled against the FIPS 140-2 compliant OpenSSL module.
+* OpenLDAP version `2.4.46`: built using openssl rather than gnutls and compiled against the FIPS 140-2 compliant OpenSSL module. 
```

Comment:
Is "openssl" a command? If not, it should be OpenSSL

Reply:
Should be OpenSSL, fixed